<launch>
  <node pkg="qualisys"
    type="qualisys_node6D"
    name="qualisys"
    output="screen">
    <param name="server_address" value="192.168.100.1"/>
    <param name="server_base_port" value="22222"/>
    <param name="publish_tf" value="false"/>
  </node>
</launch>
